docs/cep/source/source-types/stream-source.md

```diff
@@ -11,24 +11,50 @@ The stream processor groups common types of events together with a schema. This
 By default, all streams created by stream workers are sources. The syntax for defining a new stream in a stream worker is:
 
 ```sql
-CREATE STREAM <stream_name> (<attribute_name> <attribute_type>,
+   CREATE SOURCE SinkName WITH (
+      type="stream", 
+      stream.list="STRING[STRING,]", 
+      replication.type="STRING", 
+      subscription.initial.position="STRING",
+      num.consumers="INT",
+      num.listener.threads="INT",
+      num.io.threads="INT",
+      receiver.queue.size="INT",      
+      map.type='type')  
+    (<attribute_name> <attribute_type>[,
+     <attribute_name> <attribute_type>] );
+```
+
+Or you can use the syntax shortcut `CREATE STREAM`:
+
+```sql
+CREATE STREAM <stream_name> <GLOBAL | LOCAL> (<attribute_name> <attribute_type>,
```

Comment:
Do the new parameters not work for CREATE STREAM?

Reply:
There is not a way to define them with `CREATE STREAM` shorthand. So, this construct will rely on the default values.